gen-passwords.sh

```diff
@@ -4,12 +4,12 @@ function generatePassword() {
     openssl rand -hex 16
```

Comment:
It's possible to use base64 encoding to generate shorter passwords with the same entropy amount with -base64 instead of -hex.
```suggestion
    openssl rand -base64 16
```

Reply:
Could be seen as offtopic, however and maybe I should open new PR.